packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/ChangeFee/ReplaceTxButton.tsx

```diff
@@ -1,40 +1,34 @@
-import styled from 'styled-components';
-
-import { Button } from '@trezor/components';
+import { NewModal } from '@trezor/components';
+import { SelectedAccountLoaded, RbfTransactionParams } from '@suite-common/wallet-types';
 
 import { Translation } from 'src/components/suite';
 import { useDevice } from 'src/hooks/suite';
-import { useRbfContext } from 'src/hooks/wallet/useRbfForm';
-
-const Wrapper = styled.div`
-    display: flex;
-    justify-content: center;
-    margin-top: 22px;
-`;
+import { useRbf } from 'src/hooks/wallet/useRbfForm';
 
-// eslint-disable-next-line local-rules/no-override-ds-component
-const StyledButton = styled(Button)`
-    min-width: 30%;
-`;
+type ReplaceTxButtonProps = {
+    rbfParams: RbfTransactionParams;
+    selectedAccount: SelectedAccountLoaded;
+};
 
-export const ReplaceTxButton = () => {
+export const ReplaceTxButton = ({ rbfParams, selectedAccount }: ReplaceTxButtonProps) => {
```

Comment:
Unfortunately I can't use the context, since the button has been taken out of the nested content (where the context is available) and passed to NewModal bottom content instead.

Reply:
The other option would be to turn the ChangeFees component into a hook that would return the original component and the button from inside, but that seems like too much work. What do you think?

Reply:
Now that I look at it again, it's nice and clear. At first glance I saw the diff and was questioning it. Now I see that both `ChangeFee` and `ReplaceTxButton` are called from `TxDetailModal`, which passes props to both. That's normal day in React, nice and clear :+1: :ok_hand:

Trying to "unify passing props" to both components by introducing that hook would spaghettify it, so I'd be against it.